cmd/root.go

```diff
@@ -39,7 +39,7 @@ import (
 const AppName string = "forklift"
 
 // version should be in format d.d.d where d is a decimal number
-const Version string = "v0.1.23"
+const Version string = "v0.1.24"
```

Comment:
Shall it be major version change? Or at least minor?

Reply:
I've bumped up to 0.1.24 because we've always been doing that, but I agree it should be at least minor update. I will change that.